packages/next-swc/crates/next-core/src/app_segment_config.rs

```diff
@@ -279,8 +325,43 @@ fn parse_config_value(
                 return invalid_config("`preferredRegion` needs to be a static string", &value);
             };
 
-            config.referred_region = val.to_string();
+            config.referred_region = Some(val.to_string());
         }
         _ => {}
     }
 }
+
+#[turbo_tasks::function]
+pub async fn parse_segment_config_from_loader_tree(
+    loader_tree: LoaderTreeVc,
+    context: AssetContextVc,
+) -> Result<NextSegmentConfigVc> {
+    let loader_tree = loader_tree.await?;
+    let components = loader_tree.components.await?;
+    let mut config = NextSegmentConfig::default();
+    let siblings = loader_tree
```

Comment:
`loader_tree.sibling_segment_configs()`

Reply:
Also, is `sibling` the terminology used in Next.js as well? Otherwise, I'd rather we kept to `parallel`, so we know this maps to the concept of parallel routes.